docs/site/Authentication-overview.md

```diff
@@ -0,0 +1,70 @@
+---
+lang: en
+title: 'LoopBack 4 Authentication Overview'
+keywords: LoopBack 4.0, LoopBack 4, Authentication
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Authentication-overview.html
+---
+
+Security is of paramount importance when developing a web or mobile application
+and usually consists of two distinct pieces:
+
+- Authentication
+- Authorization
+
+Authentication is a process of verifying user/entity to the system, which
+enables identified/validated access to the protected routes.
+
+Authorization is a process of deciding if a user can perform an action on a
+protected resource.
+
+{% include note.html content=" For a description of an Authorization process, please see [Authorization](Loopback-component-authorization.md). " %}
+
+This document gives you an overview of the authentication system provided in
+LoopBack 4.
+
+Let's start with the following scenario: Suppose you want to secure endpoint
+`GET /todo` using a
+[JWT(JSON web token)](https://github.com/auth0/node-jsonwebtoken#readme)
+strategy. The diagram below shows how such authentication process works with
```

Comment:
This statement is a bit too technical. We can say we want to limit access to `todo` items to the owner.